pkgs/development/python-modules/py17track/default.nix

```diff
@@ -26,6 +27,26 @@ buildPythonPackage rec {
     hash = "sha256-T0Jjdu6QC8rTqZwe4cdsBbs0hQXUY6CkrImCgYwWL9o=";
   };
 
+  patches = [
+    # This patch removes references to setuptools and wheel that are no longer
+    # necessary and changes poetry to poetry-core, so that we don't need to add
+    # unnecessary nativeBuildInputs.
+    #
+    #   https://github.com/bachya/py17track/pull/80
+    #
+    (fetchpatch {
+      name = "clean-up-build-dependencies.patch";
+      url = "https://github.com/bachya/py17track/commit/3b52394759aa50c62e2a56581e30cdb94003e2f1.patch";
+      hash = "sha256-iLgklhEZ61rrdzQoO6rp1HGZcqLsqGNitwIiPNLNHQ4=";
+    })
+  ];
+
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace 'attrs = ">=19.3,<21.0"' 'attrs = ">=19.3,<22.0"' \
+      --replace 'async-timeout = "^3.0.1"' 'async-timeout = ">=3.0.1,<5.0.0"'
+  '';
```

Comment:
```suggestion
```
These no longer seem to be needed.

Reply:
Great catch! Let me take a look and see if this can be cleaned up.

Reply:
It works, and both `py17track` and `seventeentrack` (a fork from it) no longer need `async-timeout` also. I made the changes, and I'll kick off an ofborg build of those two again. Thank you for all your help reviewing.